schema/common.json

```diff
@@ -36,16 +36,55 @@
                 "name", "value"
             ]
         },
-        "Otlp": {
-            "type": "object",
+        "OtlpHttpExporter": {
+            "title": "OtlpHttpExporter",
+            "type": ["object", "null"],
             "additionalProperties": false,
             "properties": {
-                "protocol": {
-                    "type": "string",
-                    "pattern": "^(http|grpc)\\/(protobuf|json)"
+                "endpoint": {
+                    "type": ["string", "null"]
+                },
+                "certificate": {
+                    "type": ["string", "null"]
+                },
+                "client_key": {
+                    "type": ["string", "null"]
+                },
+                "client_certificate": {
+                    "type": ["string", "null"]
+                },
+                "headers": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "common.json#/$defs/NameStringValuePair"
+                    }
+                },
+                "headers_list": {
+                    "type": ["string", "null"]
+                },
+                "compression": {
+                    "type": ["string", "null"]
                 },
+                "timeout": {
+                    "type": ["integer", "null"],
+                    "minimum": 0
+                },
+                "encoding": {
+                    "type": ["string", "null"],
+                    "enum": [
+                        "proto",
+                        "json"
```

Comment:
Given that the existing options are `http/protobuf` and `http/json`, I think we should spell out `protobuf` instead of just `proto`, for the encoding enum.